micropip/transaction.py

```diff
@@ -327,6 +328,8 @@ def find_wheel(metadata: ProjectInfo, req: Requirement) -> WheelInfo:
         reverse=True,
     )
 
+    yanked_versions: list[list[WheelInfo]] = []
```

Comment:
Should this be just `list[WheelInfo]`, similar to `wheels = list(releases[ver])` below? Creating a list from a list seems to be redundant here.

```suggestion
    yanked_versions: list[WheelInfo] = []
```

Reply:
It is a list of version => list of wheels in that version.